### ISSUE TITLE
When client syncs from emtpy server db, uploading changes always results in "InvalidOperationException the serverBlob is empty"

### ISSUE THREAD

**Issue by gentledepp** [Mimetis/SyncWinRT#20]
Hi!

I have made several changes to SyncWinRT and will make a pull request soon.
One reason is that I split the nuget-package and extracted server-part into a sepearate one.
That way, I could also provide a client-package to plain .NET class libraries which we use for intgration testing.

Long story short, in our tests I noticed that we get an "InvalidOperationException serverBlob is empty" when synching changes from the client to the server.
Steps to reproduce:

1) create empty database on server
2) sync it with client
3) add row on client-side
3) sync again and you will teht this exception.

How to make it work?
1) create empty database on server
1b) Add a row on the serverside
2) sync with client

So basically if your server is still empty once your clients start syncing, you have a screwed-up database... 

Does anyone have experiences with this issue?

P.S: There is a blog-post explaining the serverBlob error:
https://jtabadero.wordpress.com/2013/01/09/synchronizing-winrtsqlite-using-sync-framework-toolkit/
It says that you have to sync with your client first, before adding any records...

**Comment by gentledepp** [comment]
this unfortunately a sync framework bug in the COM side You have to sync first, before adding any local data to avoid this error ... 👎
